src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
```

Comment:
1. Error Handling: The `throwError` method intentionally throws a `RuntimeException` with a custom message. While this can be useful for testing and review purposes, it is generally not recommended to throw generic `RuntimeExceptions` in production code. It would be better to catch specific exceptions and handle them appropriately.